crucible-mir/src/Mir/TransCustom.hs

```diff
@@ -1117,9 +1117,9 @@ discriminant_value = (["core","intrinsics", "discriminant_value"],
 
 type_id ::  (ExplodedDefId, CustomRHS)
 type_id = (["core","intrinsics", "type_id"],
-  \ _substs -> Just $ CustomOp $ \ _ _ ->
-    -- TODO: keep a map from Ty to Word64, assigning IDs on first use of each type
-    return $ MirExp knownRepr $ R.App (eBVLit (knownRepr :: NatRepr 64) 0))
+  \ _substs -> Just $ CustomOp $ \ _opTys _ops ->
+    -- TODO: keep a map from Ty to Word128, assigning IDs on first use of each type
+    return $ MirExp knownRepr $ R.App (eBVLit (knownRepr :: NatRepr 128) 0))
```

Comment:
Please leave a justification for why `128` is the right size here. Is this trying to match the size of a struct?